### PR TITLE
Sanjo/node

### PR DESCRIPTION
**Pull request by SanjoDeundiak**

### CODE REVIEW

File: implementations/rust/ockam/ockam_node/src/context.rs [hunk 1]
```diff
@@ -121,6 +107,33 @@ impl Context {
...
+    /// Send a message via a fully qualified route using specific Worker address
+    ///
+    /// Routes can be constructed from a set of [`Address`]es, or via
+    /// the [`RouteBuilder`] type.  Routes can contain middleware
+    /// router addresses, which will re-address messages that need to
+    /// be handled by specific domain workers.
+    ///
+    /// [`Address`]: ockam_core::Address
+    /// [`RouteBuilder`]: ockem_core::RouteBuilder
+    pub async fn send_message_from_address<R, M>(
```
Comment (spacekookie) [inline implementations/rust/ockam/ockam_node/src/context.rs:123]: The addition of this API endpoint is good, but should be a separate commit

**Review by spacekookie — COMMENTED** [review]
LGTM

**Comment by jared-s** [comment]
@SanjoDeundiak @spacekookie I tried to do this conflict resolution, but it is quite involved. Can you pair and help rebase this? Thanks!

**Comment by spacekookie** [comment]
@jared-s I just resolved the conflicts but I used the GitHub UI so it did a merge commit from develop. Do you want me to rebase the changes into a different PR instead?

**Comment by spacekookie** [comment]
Okay nvm I made a mistake in the web ui :see_no_evil: I rebased the changes to #1215

**Comment by jared-s** [comment]
Thank you! Merged https://github.com/ockam-network/ockam/pull/1215
